src/main/java/club/minnced/discord/webhook/send/component/ActionComponent.java

```diff
@@ -0,0 +1,16 @@
+package club.minnced.discord.webhook.send.component;
+
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Components that can be inserted inside layout components (buttons & select menus)
+ */
+public interface ActionComponent extends Component {
+
+    /**
+     * The dev-defined id of the component
+     * @return dev-defined id of the component. Nullable for link style buttons
+     */
+    @Nullable String getCustomID();
+
```

Comment:
```suggestion
     * The custom id of the component.
     * <br>This can be used for handling interactions with this component.
     *
     * @return Custom id of the component, or null for link style buttons
     */
    @Nullable
    String getCustomID();
```